src/app/shared/enums/localisations.ts

```diff
@@ -170,6 +170,9 @@ export class LocKeys {
   )
   static SPLASH_STATUS_SENDING_LOGS = new LocKeys('SPLASH_STATUS_SENDING_LOGS')
 
+  static CLINICAL_TASKS = new LocKeys('CLINICAL_TASKS')
+  static ON_DEMAND_TASKS = new LocKeys('CLINICAL_TASKS')
```

Comment:
why are these same? on demand i though will be different and will not be considered clinical tasks. Since they are not completed in clinic and thus don't need the follow-up notifications.